### PR TITLE
Add support for SV hierarchy.

### PR DESCRIPTION
**Pull request by keyurva**
* Added support for SV hierarchy (groups), description, name overrides and NL sentences.
  + Updated config doc describing these.
* Added initial unit tests.
* Added options to `run_test.sh`.

### CODE REVIEW

File: simple/sample/input/config.json [hunk 1]
```diff
@@ -19,5 +19,23 @@
...
     }
+  },
+  "variables": {
+    "var1": {
+      "name": "Good var1 name",
+      "description": "Good var1 description",
+      "nlSentences": [
+        "Natural language sentence 1",
+        "Natural language sentence 2"
+      ],
```
Comment (shifucun) [inline simple/sample/input/config.json:30]: I still think separate out the nl descriptions into a separate csv file is cleaner.

File: simple/stats/config.md [hunk 2]
```diff
@@ -17,6 +17,16 @@ The config parameters for the files to be imported should be specified in a `con
     "geoid.csv": {
       "entityType": ""
     }
+  },
+  "variables": {
```
Comment (shifucun) [inline simple/stats/config.md:21]: Can we also handle the case when variable does not have "group" field, then it is linked under "dc/g/Root" directly.

File: run_test.sh [hunk 3]
```diff
@@ -31,4 +31,64 @@ function run_lint_fix {
...
+  pip3 install -r requirements.txt
+
+  echo -e "#### Running stats tests"
+  python3 -m pytest tests/stats/ -s
+
+  deactivate
+}
+
+function run_all_tests {
+  run_lint_fix
```
Comment (shifucun) [inline run_test.sh:49]: I realized the run_list_fix should not be run in cloudbuild.yaml and used in here. This option is meant to be run locally to fix the lint issues.

We need another `run_list_test` option to replace run_list_fix in cloudbuild.yaml and here that executes commands like https://github.com/datacommonsorg/website/blob/master/run_test.sh#L108-L122

File: run_test.sh [hunk 4]
```diff
@@ -31,4 +31,64 @@ function run_lint_fix {
   deactivate
 }
 
-run_lint_fix
\ No newline at end of file
+# Fixes python tests
+function run_py_test {
```
Comment (shifucun) [inline run_test.sh:35]: thanks for adding py test, this is very good start!

File: simple/sample/input/config.json (hunk 1 above)
Comment (keyurva) [reply simple/sample/input/config.json:30]: Discussed offline to keep it here for now. Will consider moving to a different file in the future if it's helpful.

File: simple/stats/config.md (hunk 2 above)
Comment (keyurva) [reply simple/stats/config.md:21]: Done.

File: run_test.sh (hunk 3 above)
Comment (keyurva) [reply run_test.sh]: Done.

**Review by shifucun — APPROVED** [review]
thanks for the update!

**Pull request merged**